polaris-react/stylelint/plugins/coverage/configs/index.js

```diff
@@ -0,0 +1,127 @@
+module.exports = {
+  plugins: ['stylelint-scss', '../'],
+  rules: {
+    'at-rule-disallowed-list': ['keyframes'],
+    'color-named': 'never',
+    'color-no-hex': true,
+    'declaration-property-value-disallowed-list': [
+      {
+        '/color$/': ['/#[0-9a-f]{3,6}/'],
+        display: ['grid', 'flex'],
+        'font-weight': ['/[0-9]*/'],
+        opacity: ['/?![01]'],
+        'z-index': ['/[0-9]*/'],
+      },
+    ],
+    'function-disallowed-list': [
+      'brightness',
+      'contrast',
+      'hue-rotate',
+      'hsl',
+      'hsla',
+      'invert',
+      'rgb',
+      'rgba',
+      'sepia',
+    ],
+    'property-disallowed-list': [
+      'position',
+      'grid',
+      'flex',
+      'grid-row',
+      'grid-row-start',
+      'grid-row-end',
+      'grid-column',
+      'grid-column-start',
+      'grid-column-end',
+      'grid-template',
+      'grid-template-areas',
+      'grid-template-rows',
+      'grid-template-columns',
+      'grid-area',
+    ],
+    'unit-disallowed-list': ['px', 'rem', 'em', 's', 'ms'],
+    'scss/function-color-relative': true,
+    '@shopify/coverage/global-disallowed-list': [
+      [
```

Comment:
Is it possible to update the configuration of this plugin to a single array. Or perhaps convert to an object with a `patterns` key.

Reply:
Great question. Seems like only the secondary option allows for an object. It should only be a single array though–good catch! Will remove the extra `[ ... ]`'s